whois-update/src/main/java/net/ripe/db/whois/update/domain/UpdateContext.java

```diff
@@ -322,10 +325,18 @@ public void setUserSession(final UserSession userSession) {
         this.userSession = userSession;
     }
 
+    public void setOAuthSession(final OAuthSession oAuthSession) {
+        this.oAuthSession = oAuthSession;
+    }
+
     public UserSession getUserSession() {
         return userSession;
     }
 
+    public OAuthSession getoAuthSession() {
```

Comment:
Inconsistent camel-casing, can you change to `getOAuthSession` to match `setOAuthSession` ?